change/@uifabric-charting-2020-09-04-14-12-33-VerticalStackedBarChart-callout-improvements.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "major",
```

Comment:
I'm pretty sure this should NOT have been a major change. Major version bumps should be reserved for backwards-incompatible API changes, which I don't think I'm seeing in this PR.

Reply:
Never mind, talked to @Raghurk and he said this was intentional